modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -11,15 +11,13 @@ Using {product-title} with Kuryr SDN has several known limitations.
 [id="openstack-general-limitations_{context}"]
 == {rh-openstack} general limitations
 
-{product-title} with Kuryr SDN does not support `Service` objects with type `NodePort`.
+Using {product-title} with Kuryr SDN has several limitations that apply to all versions and environments:
```

Comment:
The limitations do not apply to all versions and environments.
Topic 2: Services without selectors is only from 4.7+
Topic 3: from 4.6+

Reply:
@MaysaMacedo This will be applied only to 4.7+--the other "version" limitations are to do with RHOSP versions.

Reply:
@MaysaMacedo ^^^

Reply:
Makes sense.

Reply:
+1